kits/beta-base/scripts/deploy_kit.js

```diff
@@ -57,16 +57,16 @@ module.exports = async (
     } else {
       arappFileName = 'arapp_local.json'
     }
+    if (!ensAddress) {
+      const betaArapp = require('../' + arappFileName)
+      ensAddress = betaArapp.environments[network].registry
+    }
+  } else {
+    if (!ensAddress) {
```

Comment:
You can just do an `else if {` :)

Reply:
Haha, I was actually expecting this. I did this way first, then I changed to `else if`, but then I moved back to how it is now, because it looked way better to me, because of the clear separation of the 2 use cases of the script (returning kit or not) and because of the parallelism with the previous `if` block. But if you like it better with an `else if` I can change it, of course ;-)
(I don't know what you mean by "if this should be an if block by itself")

Reply:
> (I don't know what you mean by "if this should be an if block by itself")

Just always checking if there's an `ensAddress`, since it seems like an invariant for this script to run. But we can leave it like this :).

Reply:
Oh, yes, now I get it, so you mean checking that even in when `returnKit` is `false`, leave that check outside so we also check that it was able to get it from `arapp.json` if it didn't have it before. Yes, that makes total sense, I'll change it.